Sources/Analytics/Trackers/Analytics+MultiTracker.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+#if canImport(AlicerceCore)
```

Comment:
I'm not sure I understand why we're wrapping the imports but not the code that depends on the imports, won't that fail to build? And if so, why not just let it fail on the `import` itself?

Reply:
These conditional imports are only used/relevant when we're building via SPM, where these modules actually exist and are necessary given the new sub-module dependencies specified in `Package.swift`. 

Otherwise, (e.g. via CocoaPods) these imports would break the build if not conditionally made (as the sub-modules wouldn't exist).

Also, considering our CI passed when building via both CocoaPods and SPM (which builds all modules), I'm not fully understanding your concern of the build failing... 🤔 Could you please be more specific? Or did I answer your question? 😄

Reply:
Ah, I had misunderstood the reason for these imports. Thank you for the explanation 🙇